lib/unicore/mktables

```diff
@@ -16087,7 +16087,10 @@ sub register_file_for_name($table, $directory_ref, $file) {
         # No swash means don't do the rest of this.
         next if $table->isa('Map_Table') && $table->fate != $ORDINARY;
 
-        my $sub_filename = join('/', $directory_ref->[1, -1], $file);
+        # Get the path to $file stripping off any top level directory
+        my $sub_filename = join('/',
+                                ($directory_ref->@[1, $#$directory_ref] // ""),
```

Comment:
Is $directory_ref always three elements?

If not, then I think we would want a range operator here (`1 .. $#$directory_ref`) instead?

And if it's always three elements, we could shorten this to just `$directory_ref->@[1..2]`.

Reply:
I was working too hastily; now reworked